docs/blockchain-rpc/hedera-rpc.md

```diff
@@ -0,0 +1,177 @@
+---
+description: Hedera JSON-RPC Methods
+---
+
+# Hedera
+
```

Comment:
@HGraphPunks @hgraphql LMK what you think of this intro paragraph. I feel like it could be better, but just wanted to get something up to start the discussion.

Reply:
This is good. I think maybe reword in the affirmative, something like:

"The following JSON-RPC methods are for native integration into Hedera." "Hedera also has an open-source project implementing the [Ethereum JSON-RPC standard](https://docs.hedera.com/hedera/core-concepts/smart-contracts/json-rpc-relay) which is not covered in this documentation"

I think there's opportunity in the future to test and integrate this relay into WalletConnect, to provide access to a wider audience of dApp providers that are familiar with the Ethereum JSON-RPC, relay.

Reply:
Ooh nice wording. I'll update

Reply:
Updated:
<img width="999" alt="Screenshot 2023-08-04 at 2 28 38 PM" src="https://github.com/hgraph-io/walletconnect-specs/assets/136644362/932683a4-f1d3-4656-abff-30115a1ce271">